Firestore/Swift/Tests/Codable/CodableTimestampTests.swift

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+import Foundation
+import XCTest
+
+class CodableTimestampTests: XCTestCase {
+  func testTimestampEncodes() {
+    let timestamp = Timestamp(seconds: 37, nanoseconds: 123)
+
+    let jsonData = try! JSONEncoder().encode(timestamp)
```

Comment:
From my experience using `try!` in the tests is good until it fails for some reason. Then we will see just a crash with no explanation of the reason.

I usually prefer to just wrap such operations (or entire test) into Swift `do-catch` block, e.g. :
```
func testTimestampEncodes() {
 do {
  let timestamp = Timestamp(seconds: 37, nanoseconds: 123)
  let jsonData = try JSONEncoder().encode(timestamp)
...
   XCTAssert(json.contains("\"seconds\":37"))
...
 } catch {
  XCTFail("Error: \(error)")
 }
```

Looks like you are doing this already in some tests.

Reply:
Done.